Updated files and fixed old dependencies.

trainrechs opened the pull request:
This PR contains multiple quality of life improvements, the biggest of which being:
* A removal of the contributing section of the readme, instead pointing to CONTRIBUTING.md
* Instructing Contributors to run `dotnet format` before submitting a PR
* A cleaned up CONTRIBUTING.md
* Updated Go Dependency





## Checklist

[//]: # "Put an x in [ ] to check the checkbox, like [x]"

- [ ] I discussed this pull request with Ultima or another developer before opening.
- [x ] I read the [contributing guidelines](https://www.github.com/n-Ultima/UnixBot/blob/main/.github/CONTRIBUTING.md).
- [x ] The changes in this pull request are tested.
- [x ] The changes in this pull request build(dotnet build -c Release /warnaserror) with no errors.

trainrechs committed:
Updated README.MD

Redirected contributors to CONTRIBUTING.md

trainrechs committed:
Updated CONTRIBUTING.MD

Fixed Grammar, Rephrased sentences to be more clear.

trainrechs committed:
Update CONTRIBUTING.md

trainrechs committed:
Update CONTRIBUTING.md

trainrechs committed:
Update CONTRIBUTING.md

trainrechs committed:
Update CONTRIBUTING.md

trainrechs committed:
Add Dotnet Format

trainrechs committed:
Merge pull request #1 from Voxe1/patch-2

Add Dotnet Format

trainrechs committed:
Fixed Contributing.md Link

trainrechs committed:
Merge branch 'n-Ultima:main' into main

trainrechs committed:
Updated Dependencies

Updated golang.org/x/sys

trainrechs committed:
Merge pull request #2 from Voxe1/patch-1

Updated Dependencies

trainrechs committed:
Update index.html

Grouped h1, h2, p and a together to avoid confusion

trainrechs committed:
Uncaptialize "HERE"

trainrechs committed:
Add /waranaserror

Add /warnaserror for the build script

trainrechs committed:
Merge pull request #3 from Voxe1/Voxe1-patch-1

Add /waranaserror

trainrechs committed:
Update README.md to include documentation

add the documentation to the README.

n-Ultima commented on .github/CONTRIBUTING.md:
```diff
@@ -1,20 +1,23 @@
-*  Before you contribute(via a pull request), it is asked that you run `dotnet build -c Release /warnaserror` on your local branch. We strive to not have any warnings, and this command will flag all warnings as errors. It is also asked that when naming variables, properties, etc, that they not be nonsense, and actually make sense to the context of what you're writing
-* Also, treat others how you would treat yourself. Disrespect is not tolerated. Treat other humans how you would like to be treated.
+
+*  Before you contribute(via a pull request), it is asked that you run `dotnet build -c Release /warnaserror` on your local branch. We strive to not have any warnings, and this command will flag all warnings as errors. It is also asked that when naming variables, properties, etc, that they represent the data there holding and make sense to the context of what you're writing.
```
There -> their.

n-Ultima commented on .github/CONTRIBUTING.md:
````diff
@@ -23,13 +26,18 @@ git checkout -b feature/FeatureName
 dotnet build -c Release /warnaserror
 ```
-* Commit your changes and push your changes.
+* use dotnet format, then commit and push your changes.
````
use -> Use

n-Ultima commented on readme.md:
```diff
@@ -1,6 +1,6 @@
 #  Unix Discord Bot
 ### A Moderation Utility to make moderation easier.
-This is the main source code repository for Unix. It contains the source code for the Unix Discord Bot.
+This is the main source code repository for Unix. It contains the source code for the Unix Discord Bot, as well as the eventual documentation
```
Remove the ", as well as the eventual documentation"

n-Ultima commented on readme.md:
````diff
@@ -10,28 +10,13 @@ This is the main source code repository for Unix. It contains the source code fo
-git push origin feature/MyAmazingFeature
-```
-* Open a Pull Request!
+This README is for users and server admins, You can get more details on how to contribute [here](https://github.com/n-Ultima/UnixBot/blob/main/.github/CONTRIBUTING.md).
````
I feel like this can be changed to something neater, like: "If you are interested in contributing, please view the [guidelines](https://www.github.com/n-Ultima/UnixBot/blob/main/.github/CONTRIBUTING.md).

trainrechs committed:
Merge branch 'n-Ultima:main' into main

trainrechs committed:
Updated README.md

Applied changes proposed by ultima:
Making the contributing section more concise.

trainrechs commented:
Updated README and resolved Suggestions. Have a good night!
